src/Components/WebAssembly/WebAssembly/test/Hosting/WebAssemblyCultureProviderTest.cs

```diff
@@ -24,6 +24,22 @@ public void GetCultures_ReturnsCultureClosure(string cultureName, string[] expec
         Assert.Equal(expected, actual);
     }
 
+    [Theory]
+    [InlineData("fr-FR", "tzm-Latn-DZ", new[] { "fr-FR", "fr", "tzm-Latn-DZ", "tzm-Latn", "tzm" })]
+    [InlineData("en-US", "en-GB", new[] { "en-US", "en", "en-GB" })]
+    [InlineData("fr-FR", null, new[] { "fr-FR", "fr" })]
+    [InlineData(null, null, new string[0])]
```

Comment:
After the last commit, double `null` `InlineData` can be removed, `cultureInfo` cannot be null:
`internal static string[] GetCultures(CultureInfo cultureInfo, CultureInfo? uiCultureInfo = null)`